Packs/TrustwaveFusion/README.md

```diff
@@ -0,0 +1,3 @@
+## Overview
+---
+This is the readme for the Trustwave Fusion integration.
```

Comment:
This is the readme that is shown to end users IN the marketplace on our website and should detail
1. what the pack does
2. what TW does
3. who contact for more information on your services, etc.
Include links, images, etc. to make it compelling.

Reply:
See [Pack Documentation](https://xsoar.pan.dev/docs/documentation/pack-docs) for more details.